packages/commonwealth/client/scripts/state/ui/adminOnboardingCards/adminOnboardingCards.ts

```diff
@@ -0,0 +1,69 @@
+import { createBoundedUseStore } from 'state/ui/utils';
+import { devtools } from 'zustand/middleware';
+import { createStore } from 'zustand/vanilla';
+
+interface GroupMutationBannerStore {
```

Comment:
`group` related naming is not relevant in this file, please change it everywhere